### PR TITLE
Generated Python Client Update v.2.9.39

### PR DESCRIPTION
**Pull request by ustinov**
Based on Wavefront 29.37 API Specification

Command:
```
$ curl http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.2/swagger-codegen-cli-2.4.2.jar -o ~/Applications/swagger-codegen-cli.jar

$ java -jar ~/Applications/swagger-codegen-cli.jar generate -l python \
-i https://mon.wavefront.com/api/v2/swagger.json \
-c .swagger-codegen/config.json \
--additional-properties basePath=https://YOUR_INSTANCE.wavefront.com,infoEmail=support@wavefront.com
```

Config:
```
$ cat .swagger-codegen/config.json
{
    "gitRepoId": "python-client",
    "gitUserId": "wavefrontHQ",
    "packageName": "wavefront_api_client",
    "packageUrl": "https://github.com/wavefrontHQ/python-client",
    "packageVersion": "2.9.37"
}
```

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,44 +1,45 @@
...
+    "python-dateutil>=2.1",
+    "six>=1.10",
+    "urllib3>=1.23"
+]
 
 setup(
     name=NAME,
     version=VERSION,
-    description="Wavefront Public API",
-    author="Wavefront",
```
Comment (ajayj89) [inline setup.py]: no `author` in the changed files?

Comment (ustinov) [reply setup.py]: I used to add that field manually before

File: .gitignore [hunk 2]
```diff
@@ -62,8 +62,3 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
-
-# IntelliJ
-.idea/*
-*.iml
-.DS_Store
```
[the remark this answers is not in the document]
Comment (ustinov) [reply .gitignore]: These got removed from swagger config
https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/python/gitignore.mustache

File: setup.py (hunk 1 above)
Comment (ustinov) [reply setup.py]: I'm trying to minimize the manual updates in order to be prepared for automating the job

File: .gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (ajayj89) [reply .gitignore]: Can't see those in the history as well: https://github.com/swagger-api/swagger-codegen/commits/master/modules/swagger-codegen/src/main/resources/python/gitignore.mustache

probably were not there?

[the remark this answers is not in the document]
Comment (ustinov) [reply .gitignore]: Must be entries we've been adding manually after generation

File: setup.py (hunk 1 above)
Comment (ajayj89) [reply setup.py]: that makes sense!

File: .gitignore (hunk 2 above)
[the remark this answers is not in the document]
Comment (ajayj89) [reply .gitignore]: yeah, we should keep having them then? or a way to provide our own mustache or gitingnore file :)

**Pull request merged**